onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -105,7 +106,8 @@ std::unique_ptr<RuleBasedGraphTransformer> GenerateRuleBasedGraphTransformer(Tra
 
 std::vector<std::unique_ptr<GraphTransformer>> GenerateTransformers(TransformerLevel level,
                                                                     gsl::span<const FreeDimensionOverride> free_dimension_overrides,
-                                                                    const std::vector<std::string>& transformers_and_rules_to_enable) {
+                                                                    const std::vector<std::string>& transformers_and_rules_to_enable,
+                                                                    __attribute__ ((unused)) const std::vector<std::string>& registered_execution_providers) {
```

Comment:
This won't compile with Visual Studio on the Windows side.

Reply:
Will fix